layouts/shortcodes/inline.html

```diff
@@ -0,0 +1,3 @@
+<div class="flex">
```

Comment:
wondering if we can also add in `flex-row` into the class name? that will ensure that it will always flow like a row.

we can also add things like `items-center content-center justify-center` if need be to make sure the images are centered within the div container.

Reply:
I think with this, we may not even need that flexItem shortcode in your initial comment.